src/settings/MappingProfiles/MappingProfilesTransformationsModal/MappingProfilesTransformationsModal.test.js

```diff
@@ -181,19 +181,39 @@ describe('MappingProfilesTransformationsModal', () => {
 
         describe('filling empty invalid field with invalid transformation', () => {
           beforeEach(() => {
-            userEvent.type(transformationFields[2].marcField.input, '123');
             userEvent.type(transformationFields[2].subfield.input, '12');
             userEvent.click(submitButton);
             transformationFields = getTransformationFieldGroups();
           });
 
-          it('should mark transformation field group and subfield as invalid', () => {
+          it('should mark transformation field group, marcField and subfield as invalid', () => {
```

Comment:
For consistency with code [here](https://github.com/folio-org/ui-data-export/pull/205/files#diff-7615550da86b63f77d7285393a54e4c49d5d6b2427eeea296bb3469c00bf39f3R197).
```suggestion
          it('should mark transformation field group, marc field and subfield as invalid', () => {
```